src/command/output.rs

```diff
@@ -0,0 +1,36 @@
+use atty::{self, Stream};
+
+/// RoverStdout defines all of the different types of output
+/// that come from Rover.
+#[derive(Clone, PartialEq, Debug)]
+pub enum RoverStdout {
```

Comment:
Can you mention this comment in some way here, so it's clear in code exactly what this enum/impl  is for?

> the intention here is for RoverStdout to be in charge of all machine readable data, and for all human readable data to be handled by tracing::info or tracing::warn. tracing::info will print to stderr by default, the only reason it's included here is so that we have some human readable data to go along with our machine readable data when they are both being read by humans (this info log is not printed if stdout is redirected to a file).

Reply:
yes!